app/src/main/java/br/com/alexf/boraprofut/features/game_screen/usecase/GameScreenUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package br.com.alexf.boraprofut.features.game_screen.usecase
+
+import android.util.Log
+import br.com.alexf.boraprofut.features.game_screen.mock.teamList
+import br.com.alexf.boraprofut.features.game_screen.model.ReadyMadeGames
+import kotlin.text.Typography.times
+
+class GameScreenUseCase {
+    fun getGames() : List<ReadyMadeGames> {
+        val timesEmbaralhados = teamList.shuffled()
+        val listaPartidas = mutableListOf<ReadyMadeGames>()
+        if (timesEmbaralhados.size % 2 == 0) {
+            for (i in 0 until timesEmbaralhados.size step 2) {
```

Comment:
Acho que não precisa fazer 2 step se usar a lógica de mod 2 == 0 para ser time 1 ou time 2, faz sentido?

Reply:
removi o IF e deixei o for, pois ele me da posição do ultimo item adicionado na lista e incremento com + 1 para pegar o proximo